### ISSUE TITLE
Benchmark reliability of torchbenchmarks

### ISSUE THREAD

**Issue by jerryzh168** [pytorch/benchmark#2527]
Recently I found that for the same model, the native benchmark code in torchbenchmarks does not give expected time, i.e. one is consistently slower than the other one, or one could be slower by up to 20%, I'm relying on torchao.utils.benchmark_model for now, please help take a look to see what might be the problem.

For details please see: https://github.com/pytorch/benchmark/pull/2519

**Comment by seemethere** [comment]
This seems like this is an issue with model code, our expectation is that repo owners should own model code while our team owns infrastructure.

**Comment by kit1980** [comment]
I think the time variability from run to run is expected when running on a devgpu.
TorchBench servers have some special settings to reduce the variability.

**Comment by seemethere** [comment]
> I think the time variability from run to run is expected when running on a devgpu. TorchBench servers have some special settings to reduce the variability.

Oh so is this more of an infrastructure thing?

**Comment by jerryzh168** [comment]
yeah this is infrastructure thing I think, since with the exact same setup `torchao.utils.benchmark_model` gives stable results
